package.json

```diff
@@ -9,6 +9,10 @@
     "email": "",
     "url": "http://aws.amazon.com/"
   },
+  "repository": {
```

Comment:
Elaborate on why this change is needed in the Description section. 
Have built the package successfully, if so make a note of this in the Testing section.